app/views/waste_exemptions_engine/applicant_name_forms/new.html.erb

```diff
@@ -8,25 +8,25 @@
 
     <p><%= t(".description") %></p>
 
-    <div class="form-group <%= "form-group-error" if @applicant_name_form.errors[:first_name].any? %>">
-      <fieldset id="first_name">
-        <% if @applicant_name_form.errors[:first_name].any? %>
-            <span class="error-message"><%= @applicant_name_form.errors[:first_name].join(", ") %></span>
+    <div class="form-group <%= "form-group-error" if @applicant_name_form.errors[:applicant_first_name].any? %>">
+      <fieldset id="applicant_first_name">
+        <% if @applicant_name_form.errors[:applicant_first_name].any? %>
+            <span class="error-message"><%= @applicant_name_form.errors[:applicant_first_name].join(", ") %></span>
         <% end %>
 
-        <%= f.label :first_name, t(".first_name_label"), class: "form-label" %>
-        <%= f.text_field :first_name, value: @applicant_name_form.first_name, class: "form-control govuk-input--width-20" %>
+        <%= f.label :applicant_first_name, t(".first_name_label"), class: "form-label" %>
```

Comment:
Can we update the I18n bits while we're at it? (eg `.applicant_first_name_label`) Will just bring everything into line then :)

Reply:
Sounds good :)